test/sql/updates/setup.policies.sql

```diff
@@ -20,21 +20,32 @@ SELECT
 
 DO LANGUAGE PLPGSQL $$
 DECLARE
-  ts_version TEXT;
+  ts_major INT;
+  ts_minor INT;
 BEGIN
 
-  SELECT extversion INTO ts_version FROM pg_extension WHERE extname = 'timescaledb';
+  SELECT
+    (string_to_array(extversion,'.'))[1] INTO ts_major
+    FROM pg_extension WHERE extname = 'timescaledb';
+
+  SELECT
+    (string_to_array(extversion,'.'))[2] INTO ts_minor
+    FROM pg_extension WHERE extname = 'timescaledb';
```

Comment:
```suggestion
WITH timescale_version AS (
    SELECT string_to_array(extversion,'.') AS v
    FROM pg_extension
    WHERE extname = 'timescaledb'
)
SELECT v[1], v[2]
INTO ts_major, ts_minor
FROM timescale_version;
```
Nit... just more elegant SQL :-)

Reply:
Thanks, this looks much cleaner than my SQL statement.